src/polus/plugins/__init__.py

```diff
@@ -1,2 +1,38 @@
-# This file should not be changed
-from . import *  # noqa: F403,F401
+"""Initialize polus-plugins module."""
+
+import logging
+
+from polus.plugins._plugins.classes import (  # noqa # pylint: disable=unused-import
+    get_plugin, list_plugins, load_plugin, refresh)
+from polus.plugins._plugins.update import (  # noqa # pylint: disable=unused-import
+    update_nist_plugins, update_polus_plugins)
+
+"""
+Set up logging for the module
+"""
+logging.basicConfig(
+    format="%(asctime)s - %(name)-8s - %(levelname)-8s - %(message)s",
+    datefmt="%d-%b-%y %H:%M:%S",
+)
+logger = logging.getLogger("polus.plugins")
+refresh()  # calls the refresh method when library is imported
+__plugins = list_plugins()
+
+for _p in __plugins:
+    # make each plugin available as polus.plugins.PluginName
+    globals()[_p] = get_plugin(_p)
```

Comment:
I think another way of doing is to use the module level attributes.

https://github.com/Nicholas-Schaub/neurofilament/blob/9397f8ba56200268149a1679f1d03976a844f0af/neurofilament/data/__init__.py#L139

The one thing I haven't figured out is how to make sure that we enable type hinting for any of this.

Reply:
Swithced to `__getattr__`